packages/api-client/src/helpers/magentoLink/graphQl.ts

```diff
@@ -26,27 +28,28 @@ const createErrorHandler = () =>
 
         if (!message.includes("Resource Owner Password Credentials Grant")) {
           if (!locations) {
-            consola.error(`[GraphQL error]: Message: ${message}, Path: ${path}`);
+            logger.error(`[GraphQL error]: Message: ${message}, Path: ${path}`);
             return;
           }
 
           const parsedLocations = locations.map(({ column, line }) => `[column: ${column}, line: ${line}]`);
 
-          consola.error(`[GraphQL error]: Message: ${message}, Location: ${parsedLocations.join(", ")}, Path: ${path}`);
+          logger.error(`[GraphQL error]: Message: ${message}, Location: ${parsedLocations.join(", ")}, Path: ${path}`);
         }
       });
     }
 
     if (networkError) {
-      consola.error(`[Network error]: ${networkError}`);
+      logger.error(`[Network error]: ${networkError}`);
```

Comment:
Network error can be a metadata kind. the network error can be logged directly
```suggestion
      logger.error(networkError, { type: 'Network Error'});
```
The type is just a proposition. We do not have convention here.